README.md

```diff
@@ -44,7 +44,12 @@ echo (new SqlFormatter())->format($query);
 
 Output:
 
-![](http://jdorn.github.com/sql-formatter/format-highlight.png)
+<img src="examples/readme_format_html.svg" width="600" height="450" alt="formatted output with HTML Highlight">
```

Comment:
To enable HTML style on Github README.md,  I chose this hack.
https://css-tricks.com/custom-styles-in-github-readmes/